client/app/services/service-details/service-details.component.js

```diff
@@ -30,10 +30,10 @@ function ComponentController($stateParams, $state, $window, CollectionsApi, Even
       availableTags: [],
       credential: {},
       listActions: [],
-
       // Functions
       hasCustomButtons: hasCustomButtons,
       disableStopButton: disableStopButton,
+      displayedContent: 'details',
```

Comment:
This doesnt need that, this merely is a control for which active screen of information someone is looking at.